.github/workflows/deploy.yml

```
@@ -0,0 +1,106 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+env:
+  APP_NAME: idlemmo-conquest
+  DEPLOY_PATH: /home/${{ secrets.VPS_USER }}/${{ env.APP_NAME }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)